.github/workflows/tests.yml

```diff
@@ -313,36 +284,33 @@ jobs:
       - uses: actions/setup-python@v4
         with:
           python-version: ${{ env.PYTHON_VERSION }}
-      - uses: abatilo/actions-poetry@v2
-        with:
-          poetry-version: ${{ env.POETRY_VERSION }}
-      - uses: arduino/setup-protoc@v2
-        with:
-          version: ${{ env.PROTOC_VERSION }}
-          repo-token: ${{ secrets.GITHUB_TOKEN }}
-      - uses: actions/setup-node@v4
-        with:
-          node-version: ${{ env.NODE_VERSION }}
       - uses: actions/cache@v3
```

Comment:
I believe we shall remove all the caching except for rust : python, deno, node do not take that long and that will let us use the whole 10Go for ghjk internal cache, and an explicit cache on the rust target(s)